WordPress/Classes/Services/BlogService.m

```diff
@@ -590,6 +590,37 @@ - (void)removeBlog:(Blog *)blog
     [WPAnalytics refreshMetadata];
 }
 
+- (void)associateSyncedBlogsToJetpackAccount:(WPAccount *)account
+                                     success:(void (^)(void))success
+                                     failure:(void (^)(NSError *error))failure
+{
```

Comment:
Just making a note here that there might be some overlap with `migrateRemoteJetpackBlog:forAccount:`.

This method is currently only associating the existing `Blog` with the new `WPAccount` if it's contained in the `me/sites` response. I'd suggest, since we just retrieved the data, that we update the matching blog with `updateBlogWithRemoteBlog:`, which in turn would call the migration.

I don't see any immediate problems if we don't and I'll keep adding on this as part of the connection flow, so feel free to skip the suggestion.

Reply:
I'll leave it to you then :)